pkg/backend/display/progress.go

```diff
@@ -812,8 +812,8 @@ func (display *ProgressDisplay) processNormalEvent(event engine.Event) {
 	// If we're suppressing reads from the tree-view, then convert notifications about reads into
 	// ephemeral messages that will go into the info column.
 	if metadata != nil && !display.opts.ShowReads {
-		if metadata.Op == deploy.OpReadDiscard || metadata.Op == deploy.OpReadReplacement {
-			// just flat out ignore read discards/replace.  They're only relevant in the context of
+		if metadata.Op == deploy.OpReadReplacement {
+			// just flat out ignore read replace.  They're only relevant in the context of
```

Comment:
I think that this is going to cause rendering changes for discards of read resources--these discards will now be shown where they were previously hidden. Might be worth reimplementing this filter in terms of `OpDiscard` on an external resource.